lsp-mode.el

```diff
@@ -7960,8 +7960,8 @@ Returns nil if the project should not be added to the current SESSION."
           (t nil)))
     ('quit)))
 
-(declare-function tramp-file-name-host "tramp" (file))
-(declare-function tramp-dissect-file-name "tramp" (file))
+(declare-function tramp-file-name-host "tramp" (file) t)
```

Comment:
I think we should also prefix `tramp` with `ext:`.
A simple reason is that it works even when `tramp.el` is not available.
It's true that `tramp.el` is included in Emacs, but it's completely out of this package control, and just simple add `ext:` every where provides a more future proof way.

Reply:
OK, I prefixed the `tramp` and `org` stuff with `ext:`.

I think another reason to use `ext:` here is when we depend on a version of `tramp` or `org` that is newer than what ships with the bundled Emacs.

It's weird to me that no one else does this for `tramp` or `org` though. Even `tramp` eschews `ext:` when using `declare-function` for `org` functions.